ci-operator/config/openshift/hypershift/openshift-hypershift-release-4.14__periodics.yaml

```diff
@@ -151,6 +151,13 @@ tests:
       MCE_VERSION: "2.3"
       ODF_OPERATOR_SUB_CHANNEL: stable-4.13
     workflow: hypershift-mce-kubevirt-baremetalds-conformance
+- as: e2e-mce-power-conformance
+  cron: 0 4 * * *
```

Comment:
@Shilpa-Gokul @tvardema Can we review this frequency and ensure we set it to a healthy level of runs+reviews.